ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -819,4 +819,94 @@
               case-sensitive: true
               default-excludes: true
               fingerprint: false
-              only-if-success: false
\ No newline at end of file
+              only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-e2e
+          node: 'antrea-kind-testbed'
+          description: 'This is for running e2e tests on kind.'
+          branches:
+          - '*/main'
```

Comment:
This should be set to `- ${{sha1}}`. Setting it to main means testing commit from the latest main branch. Setting to `- ${{sha1}}` means testing commit from PR. Same comment for all other branches setting in the newly added code.